.github/workflows/sth-build-test.yml

```
@@ -326,3 +326,122 @@ jobs:
       - name: Run BDD tests
         run: SCRAMJET_TEST_LOG=1 yarn test:bdd-ci
 
+  test-bdd-ci-no-host-no-docker-sth:
+    name: Test bdd-ci-no-host-no-docker STH (Nodejs ${{ matrix.node-version }})
+    needs: [analyze-code, build-sth, build-refapps, build-docker-sth-image, build-docker-runner-image, build-docker-pre-runner-image]
```

So, in this case we don't need `build-docker-runner-image` and `build-docker-pre-runner-image`

```suggestion
    needs: [analyze-code, build-sth, build-refapps, build-docker-sth-image]
```

---

fixed